compiler/testData/codegen/bytecodeText/companion/kt14258.kt

```diff
@@ -0,0 +1,19 @@
+// Checks that methods 'access$getMy$p', 'access$getMy$cp' and 'getMy' are not generated and
+// that backed field 'my' is directly used through a 'getstatic'
+
+class My {
+    companion object {
+        private val my: String = "OK"
+    }
+
+    fun getMyValue() = my
+}
+
+fun box() = My().getMyValue()
```

Comment:
`fun box()` is not needed in bytecode text tests

Reply:
`fun box()` removed